### PR TITLE
Added works and about myself sections for mobile version

### PR DESCRIPTION
**Pull request by Kakalanp**
Almost everything is inside one commit because GitHub didn't allow me to commit before as I've been dealing with Internet problems. Still, being so close to the deadline of the project, I can't afford waiting times.

Adding the works part of the mobile version, and it's respective CSS.

Adding the About part of the mobile version, and it's respective CSS.

Adding icons for the About part, and it's respective background image.

Fixed some minor formatting problems I left on yesterday's version (Just to make easier the transition to desktop when time comes, I hope)

### CODE REVIEW

**Comment by Kakalanp** [comment]
To my reviewer: Hello and thank you for your time.
As I stated on the pull description, I'm at the edge of a deadline, so, if there's anything I need to fix, please be as specific as possible, so I can catch up to the program time estimates, thank you very much, in advance.

File: styles.css [hunk 1]
```diff
@@ -49,10 +61,94 @@ h1 {
...
 .social {
   flex: 1;
-  list-style: none;
+}
+
+.works {
+  padding: 0% 10%;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (FrederickMih) [inline styles.css]: - [x] From the requirements of this project, the main container of the project section should have a `display:grid`. Please let's rectify this issue. In this project, we are required to make use of both CSS Grid and Flexbox for layouting.

File: index.html [hunk 2]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+                <a class="button" href="">See project</a>
```
Comment (FrederickMih) [inline index.html:88]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 3]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
```
Comment (FrederickMih) [inline index.html:105]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 4]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
```
Comment (FrederickMih) [inline index.html:123]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 5]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
```
Comment (FrederickMih) [inline index.html:142]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 6]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
```
Comment (FrederickMih) [inline index.html:160]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 7]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
+
     </section>
+
+
+    <section class="about">
+        <section class="aboutme">
+            <h1>About me</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+             <a class="button" href="">Get My Resume</a>
```
Comment (FrederickMih) [inline index.html:172]: - [ ] In order to make our button more user-friendly, please include the Active, Hover, and Pressed styling effects for this button.

File: index.html [hunk 8]
```diff
@@ -41,7 +46,164 @@ <h2 class="enphasis">I'm a software developer</h2>
...
+            <h3>Languages</h3>
+            <ul class="tags">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>codepen</li>
+                <li>Gitlab</li>
+                <li>Termial</li>
+            </ul>
+        </section>
+    </section>
```
Comment (FrederickMih) [inline index.html]: - [ ] The list of social media icons are currently missing in your about me section. Please let's make sure we include them. Social media icons reinforce a relationship between your portfolio and the update you make on your social media pages hence useful for SEO.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
### Hi @Kakalanp , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight

:arrow_forward: There is no linter errors:heavy_check_mark: 
:arrow_forward: Github flow best practices have been followed:heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (Kakalanp) [reply index.html]: The preset I used doesn't have social media icons until the contact form, which is not in the page yet.

File: styles.css (hunk 1 above)
Comment (Kakalanp) [reply styles.css]: I did use a `display:grid` for the main section, I will, either way, put it in there.

File: index.html [hunk 9]
```diff
@@ -41,7 +46,169 @@ <h2 class="enphasis">I'm a software developer</h2>
...
+        <section class="card">
+            <img src="/Images/languages-icon.svg" alt="languages">
+            <h3>Languages</h3>
+            <ul class="tags">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+        </section>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] [OPTIONAL] Kindly fix the image link as it looks broken
      your design 👇 
      
image
    Figma design 👇 
       
image

     Kindly fix for all your card classes

Comment (VICTOR-CHUKWU) [inline index.html]: # Optional but Important
- [x] [OPTIONAL] According to project requirement your card `section` should be `ul`  element and `framework`, `skills` and `language` should be wrapped in `li` element, kindly fix that

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Kakalanp  👏 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (Kakalanp) [reply index.html]: I don't see them broken no matter what I do, any clues on why or how to solve that?

Comment (VICTOR-CHUKWU) [reply index.html]: -  Try adding a correct path maybe you can name the path like this `src='./Images/Languages-icon.svg'` also try to get a live link of your project by hosting with GitHub so you can know how your work looks when live

Comment (Kakalanp) [reply index.html]: Thanks for that! I completely forgot about it.

**Pull request merged**